modules/rest-util/test/blaze/middleware/fhir/resource_test.clj

```diff
@@ -193,6 +205,41 @@
               :body (input-stream (str "<Binary xmlns=\"http://hl7.org/fhir\"><data value=\"" (apply str (repeat (* 8 1024 1024) \a)) "\"/></Binary>"))})
       fhir-spec/fhir-type := :fhir/Binary)))
 
+(deftest binary-test
+  (testing "when sending FHIR-wrapped Binary data"
+    (testing "both handlers should return the same for both wrappers (JSON and XML)"
+      (let [b64-encoded-binary-data "MTA1NjE0Cg==" ;; raw data: 105614
+            binary-resource-data (type/base64Binary b64-encoded-binary-data)
+            binary-resource-content-type (type/code "text/plain")]
```

Comment:
content type should be in string form here not FHIR code, because you use it in string concatenation, just wrap it into a FHIR code later in the given block